src/api/consumer/lib/untrack.js

```diff
@@ -20,11 +21,14 @@ export default (async function untrack(componentIds: string[]): Promise<Object>
     return { untrackedComponents: newComponents, unRemovableComponents, missingComponents: missing };
   }
   componentIds.forEach((componentId) => {
-    if (includes(newComponents, componentId)) {
-      untrackedComponents.push(componentId);
-      bitMap.removeComponent(componentId);
+    const bitId = BitId.parse(componentId);
```

Comment:
If you use toString() in all places just store the toString result.
```
resolvedComponentId = BitId.parse(componentId).toString();
```
and use this in all places

Also please make a comment about it, no one will understand why you did it and someone will delete in by accident in the future